prob44/prob44.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <script type="text/javascript">
+      const calculate=(array)=> {
+        var stats=[];
+        var smallest = array[0];
+        var largest = array[0];
+        var sum = 0;
+        for (var x = 0; x < array.length; x++) {
+          if (smallest>array[x]) {
+            smallest = array[x];
+          }
+          if (largest<array[x]) {
+            largest = array[x];
+          }
+          sum+=array[x];
+          var average = sum/array.length
```

Comment:
Take a look at this variable and consider the scoping of it.

Reply:
ok